app/routes.js

```diff
@@ -68,6 +68,7 @@ export const routes = (authProvider, sessionManager) => {
   router.get('/organisation/:orderId/preview', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
     const { orderId } = req.params;
+    const { print } = req.query;
```

Comment:
Can print equal anything on the query string? 

For example:
`?print=fawadisgreat?`

This would still render the print template?